src/pluginVanilla/java/mcp/mobius/waila/plugin/vanilla/provider/PlantProvider.java

```diff
@@ -31,6 +35,31 @@ private static void addMaturityTooltip(ITooltip tooltip, float growthValue) {
         }
     }
 
+    private static void addCropGrowableTooltip(ITooltip tooltip, IBlockAccessor accessor) {
+        int lightLevel = accessor.getWorld().getRawBrightness(accessor.getPosition(), 0);
+        tooltip.addLine(new PairComponent(
+            Component.translatable(Tl.Tooltip.CROP_GROWABLE), lightLevel >= 9
```

Comment:
```suggestion
        tooltip.addLine(new PairComponent(Component.translatable(Tl.Tooltip.CROP_GROWABLE), lightLevel >= 9
```

This should also be applied to other instances of this.